### PR TITLE
Add amendment for world merge-include

### PR DESCRIPTION
**Pull request by azeey**
# 🎉 New feature

## Summary
Extends model merge-include to worlds. The merge mechanism should be exactly the same except the type of elements that are allowed to be merged.

[Preview](http://sdformat.org/tutorials?tut=composition_merge_proposal&cat=pose_semantics_docs&branch=azeey%2Fworld_merge_include)

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by aaronchongth — APPROVED** [review]
LGTM!

File: composition/merge_proposal.md [hunk 1]
```diff
@@ -123,6 +137,21 @@ to the included model's `__model__` frame or to use
...
+Merge-include in `<world>` would allow models that themselves contain nested
+models to be merged into the world such that the nested models are placed
+directly in the `<world>` without the additional name scope of the parent
+model. The mechanism for merging works the same way as for models except that
+links cannot be merged into the world since `//world/link` is not a valid
+SDFormat element. Note: as of [libsdformat
+13.x](https://github.com/gazebosim/sdformat/pull/1117), `//world/joint` is
+included in the spec, thus `//model/joint` is allowed. The parser should emit
+errors if it encounters forbidden elements while trying to merge-include models
+into the world.
```
[the remark this answers is not in the document]
Comment (azeey) [reply composition/merge_proposal.md:154]: Added in https://github.com/gazebosim/sdf_tutorials/pull/86/commits/6f79a8a74354b0e4fe2d162ec74db9d42b35d198. Also mentioned that `//model/grippers` are forbidden.

File: composition/merge_proposal.md [hunk 2]
```diff
@@ -123,6 +137,22 @@ to the included model's `__model__` frame or to use
...
+### `//world/include/@merge`
+
+**Amendment**: This section has been added as part of Amendment 1.
+
+Merge-include in `<world>` would allow models that themselves contain nested
+models to be merged into the world such that the nested models are placed
+directly in the `<world>` without the additional name scope of the parent
+model. The mechanism for merging works the same way as for models except that
+links and grippers cannot be merged into the world since `//world/link` and 
+`//world/gripper` are  not a valid SDFormat elements. Note: as of [libsdformat
```
Comment (scpeters) [inline composition/merge_proposal.md:149]:
```suggestion
`//world/gripper` are not valid SDFormat elements. Note: as of [libsdformat
```

**Review by scpeters — APPROVED** [review]
LGTM after one more grammar change

File: composition/merge_proposal.md (hunk 2 above)
Comment (azeey) [reply composition/merge_proposal.md]: https://github.com/gazebosim/sdf_tutorials/pull/86/commits/e5a28277e86dc10c959a3bbcdb99e61eb4490b72

**Pull request merged**